File: 2800~2899/2839.c
#include <stdio.h>
int main() {
	int input;
	scanf("%d", &input);
	int three = 0;
	while (1) {
		if (input % 5 == 0) {
			printf("%d", input / 5 + three);
			break;
		}
		else if (input <= 0) {
			printf("-1");
			break;
		}
		else {
			input -= 3;
			three++;
		}
	}
}

File: 2800~2899/2839.py
input = int(input())
three = 0
while (1):
    if (input % 5 == 0):
        print("%d" %(input / 5 + three))
        break
    elif (input <= 0):
        print("-1")
        break
    else:
        input -= 3
        three+=1
